packages/gitbook-docs/src/markdoc/schema/link.ts

```diff
@@ -47,6 +47,15 @@ export const link: Schema = {
       : titleFromAnchor
       ? [titleFromAnchor]
       : childrenTreeNode;
+
+    if (attrs.href.match(/\/#/)) {
+      const splitted = attrs.href.split('/');
+      const anchor = splitted[splitted.length - 1].replace('/', '');
+      const href = splitted.slice(0, -1).join('/');
+      const linkAttrs = { ...attrs, href: `${href}${anchor}` };
+      return new Tag('Link', linkAttrs, children);
```

Comment:
I would consider putting this login inside [LinkAttr](https://github.com/pagopa/developer-portal/blob/558d23c/packages/gitbook-docs/src/markdoc/attributes.ts#L27-L68)

Reply:
I moved the logic into the [convertLink](https://github.com/pagopa/developer-portal/blob/558d23c/packages/gitbook-docs/src/markdoc/attributes.ts#L19) function .